share/doc/wake/basic_wake_tutorial.md

```diff
@@ -0,0 +1,660 @@
+## Wake Tutorial
+
+### Invoking wake
+
+1. Create a wake database: `wake --init .`
+
+2. Run command for wake: `wake -vx '<function_name> <parameter_if_any>'`
+
+3. To know about various wake options run `wake --help`. `v` stands for verbosity and `x` stands for execute the expression.
+
+4. Create a file with `.wake` extension and copy the functions mentioned in this tutorial and run them using `wake` command in the terminal.  
+
+5. For more information on wake library click [here](https://sifive.github.io/wake/)
+
+### wake `Hello World`
+* If you want to display a string you can do that using the below function by writing the string to be displayed inside "" and call the function abc . Ex: `wake -x 'abc'`. This will call the function `abc` and prints `Hello World`. 
+```
+def abc = "Hello World"
+```
```

Comment:
This would be more tutorial like. Tutorial needs to consist of commands that work when copy-pasted EXACTLY.
Is the below correct? I don't even know reading the text as currently written.

### wake `Hello World`
Put the following into a file called `helloworld.wake`:

```
def abc = "Hello World"
```

Then execute the following command:
```
wake -x 'abc'
``` 

This will call the function `abc` and print `Hello World`.

Reply:
I have added the explanation in the similar pattern